front/app/containers/Authentication/useSteps/index.ts

```diff
@@ -36,13 +36,15 @@ import {
 } from '../typings';
 import { SSOParams } from 'services/singleSignOn';
 import { isNilOrError } from 'utils/helperUtils';
+import useUpdateUser from 'api/users/useUpdateUser';
 
 let initialized = false;
 
 export default function useSteps() {
   const anySSOEnabled = useAnySSOEnabled();
   const { pathname, search } = useLocation();
   const authUser = useAuthUser();
+  const { mutate: updateUser } = useUpdateUser();
```

Comment:
In the missingDataFlow component, we originally awaited the update service user - should we use mutateAsync in this case then? :thinking:

Reply:
Yes, exactly.